plugins/modules/device_configs_backup_workflow_manager.py

```diff
@@ -339,6 +397,74 @@ def get_device_ids_list(self):
         self.log("Backup of {0} devices out of {1} devices is possible".format(valid_device_count, original_valid_device_count), "INFO")
         return device_ids
 
+    def get_site_devices(self):
+        """
+        Fetches the list of device ids when site is passed along with other parameters
+        Args:
+            self: The instance of the class containing the 'config' attribute to be validated.
+        Returns:
+            device_in_site: The list of device ids based on the parameters passed by the user,
```

Comment:
List of device IDs that match the criteria and are located at the specified site.

Reply:
Done